recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "0.20.0" %}
+{% set version = "0.21.0" %}
 
 package:
   name: evalml-core
   version: '{{ version }}'
```

Comment:
@freddyaboulton are these quotes necessary? I see the bot removes them in every PR, and I notice [woodwork doesn't have them](https://github.com/conda-forge/woodwork-feedstock/blob/master/recipe/meta.yaml).

Reply:
Yep we need them for the build_conda ci job!